appcelerator-archive/amp
# Remove references to stats or metrics

Closes #1728 

#### To test

Check Travis build.

Also check Kibana at http://kibana.local.appcelerator.io/ for only logs data.
